exasol_integration_test_docker_environment/doctor.py

```diff
@@ -78,14 +78,14 @@ def is_supported_platform() -> bool:
     return sys.platform in SUPPORTED_PLATFORMS
 
 
-def health_checkup() -> Iterable[error.ExaError]:
+def health_checkup() -> Iterable[Error]:
     """
     Runs all known examinations
 
     return an iterator of error codes specifying which problems have been identified.
     """
     check_function = Callable[[], bool]
-    diagnosis_function = Callable[[], Iterable[error.ExaError]]
+    diagnosis_function = Callable[[], Iterable[Error]]
```

Comment:
I have the feeling we should rename Error to HealthProblem or something to distingish it from ExaError

Reply:
Good idea

Reply:
Done